src/Symfony/Component/Console/Formatter/LexerInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Formatter;
+
+/**
+ * @author Krisztián Ferenczi <ferenczi.krisztian@gmail.com>
+ */
+interface LexerInterface
+{
+    /**
+     * Tokenize a text.
+     *
+     * @param string $text
+     *
+     * @return \IteratorAggregate
```

Comment:
should return `\Traversable` or `iterable`. There is no reason to enforce that `\IteratorAggregate` is used to implementator the iterator.